packages/kbn-securitysolution-utils/src/path_validations/index.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const FILENAME_WILDCARD_WARNING = i18n.translate('utils.filename.wildcardWarning', {
+  defaultMessage: `A wildcard in the filename will affect the endpoint's performance`,
+});
+
+export const FILEPATH_WARNING = i18n.translate('utils.filename.pathWarning', {
+  defaultMessage: `Path may be formed incorrectly; verify value`,
+});
+
+export enum ConditionEntryField {
```

Comment:
Other might want to comments, but I would vote for not using `enum`s

Reply:
Why? What would you suggest we use instead? 🤔

Reply:
lets wait to hear from others (I would like to hear @FrankHassanabad 's opinion here). I would just go with a regular `type` and maybe create and/or `const`'s for the runtime values.

Reply:
I see.

Reply:
For the enum's, I would at least do a `export enum ConditionEntryField` if you can. You don't have to use `enum` also and I know there's been some discussions but so far it seems like people are fine with `const enum` patterns. We don't enforce anything at the linter level at the moment.

I do want to hear from people if they want to completely remove `enum` or avoid it for reasons so I know more but I'm not strong for or against enum's I would say. The `const enum` seems to be better.